File: 프로그래머스/0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.py
def solution(n):
    odd = 0
    even = 0
    if n % 2 != 0:
        for i in range(0, n + 1):
            # odd += i
            if i % 2 != 0:
                odd += i
        return odd
    else:
        for i in range(0, n + 1):
            if i % 2 == 0:
                even += (i * i)
        return even

File: 프로그래머스/0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.js
function solution(n) {
    var answer = 0;
    if (n % 2 !== 0) {
        for (let i = 1; i <= n; i++) {
            if (i % 2 !== 0) {
                answer += i;
                console.log(answer);
            }
        }
    } else {
         for (let i = 1; i <= n; i++) {
            if (i % 2 === 0) {
                answer += i * i;
            }
        }
    }
    return answer;
}